pkg/tls/tls.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/openshift/library-go/pkg/crypto"
 )
 
+const DefaultCertificateDuration = 825 * 24 * time.Hour
```

Comment:
is this true for all secrets? Or just those signed by our CSR controller?

Reply:
Currently, this is true of all TLS secrets we create, other than the CA itself.